deploy-agent/deployd/agent.py

```diff
@@ -428,6 +492,10 @@ def main():
     else:
         agent.serve_once()
 
+    create_sc_gauge('deployd.stats.internal.time_elapsed_proc_sec',
```

Comment:
this looks like a timer so it should be a timer not a gauge.

Reply:
fixed, thank you.